src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/Objection.java

```diff
@@ -90,6 +103,10 @@ public Objection build() {
     @Field("http_request_id")
     @JsonIgnore
     private String httpRequestId;
+    @Field("full_name")
+    private String fullName;
+    @Field("share_identity")
+    private boolean shareIdentity;
```

Comment:
I would set this as a Boolean, so if the value is somehow not present in the db, we get null instead of false

Reply:
What we want here is probably a default of false.